### PR TITLE
add nmt models for en-de pair

### PR DESCRIPTION
**Pull request by bes-dev**
Re-open https://github.com/openvinotoolkit/open_model_zoo/pull/2096 with fixed merge conflicts

### CODE REVIEW

**Comment by bes-dev** [comment]
@eaidova @Anna-Mironova it is reopened version of the previous PR with fixed merge conflicts. Please look at PR

**Comment by Anna-Mironova** [comment]
Jenkins ok to test

**Comment by Anna-Mironova** [comment]
@bes-dev please add new line to the end of following files:

13:07:35  models/intel/machine-translation-nar-de-en-0001/tokenizer_src/vocab.json:1: last line doesn't end with a newline character
13:07:35  models/intel/machine-translation-nar-de-en-0001/tokenizer_tgt/vocab.json:1: last line doesn't end with a newline character
13:07:35  models/intel/machine-translation-nar-en-de-0001/tokenizer_src/vocab.json:1: last line doesn't end with a newline character
13:07:35  models/intel/machine-translation-nar-en-de-0001/tokenizer_tgt/vocab.json:1: last line doesn't end with a newline character

**Comment by bes-dev** [comment]
> @bes-dev please add new line to the end of following files:
> 
> 13:07:35 models/intel/machine-translation-nar-de-en-0001/tokenizer_src/vocab.json:1: last line doesn't end with a newline character
> 13:07:35 models/intel/machine-translation-nar-de-en-0001/tokenizer_tgt/vocab.json:1: last line doesn't end with a newline character
> 13:07:35 models/intel/machine-translation-nar-en-de-0001/tokenizer_src/vocab.json:1: last line doesn't end with a newline character
> 13:07:35 models/intel/machine-translation-nar-en-de-0001/tokenizer_tgt/vocab.json:1: last line doesn't end with a newline character

done

**Review by vladimir-dudnik — APPROVED** [review]
looks ok for me

File: models/intel/machine-translation-nar-en-de-0001/description/machine-translation-nar-en-de-0001.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+name: tokens
+shape: `1, 150`
+description: sequence of tokens (integer values) representing the tokenized sentence.
+The sequence structure is as follows (`<s>`, `</s>` and `<pad>` should be replaced by corresponding token IDs as specified by the dictionary):
+`<s>` + *tokenized sentence* + `</s>` + (`<pad>` tokens to pad to the maximum sequence length of 150)
+
+## Output
+
+name: preds
```
Comment (eaidova) [inline models/intel/machine-translation-nar-en-de-0001/description/machine-translation-nar-en-de-0001.md:37]:
```suggestion
name: `preds`
```
```suggestion
name: preds
```

File: models/intel/machine-translation-nar-en-de-0001/description/machine-translation-nar-en-de-0001.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+The quality metrics were calculated on the wmt19-en-de dataset ("test" split in lower case).
+
+| Metric                    | Value         |
+|---------------------------|---------------|
+| BLEU                      |        17.7 % |
+
+
+## Input
+
+name: tokens
```
Comment (eaidova) [inline models/intel/machine-translation-nar-en-de-0001/description/machine-translation-nar-en-de-0001.md:29]:
```suggestion
name: `tokens`
```
```suggestion
name: tokens
```

**Review by eaidova — APPROVED** [review]

File: models/intel/machine-translation-nar-de-en-0001/description/machine-translation-nar-de-en-0001.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+The quality metrics were calculated on the wmt19-en-de dataset ("test" split in lower case).
+
+| Metric                    | Value         |
+|---------------------------|---------------|
+| BLEU                      |        21.4 % |
+
+
+## Input
+
+name: tokens
```
Comment (eaidova) [inline models/intel/machine-translation-nar-de-en-0001/description/machine-translation-nar-de-en-0001.md:29]:
```suggestion
name: `tokens`
```

File: models/intel/machine-translation-nar-de-en-0001/description/machine-translation-nar-de-en-0001.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+shape: `1, 150`
+description: sequence of tokens (integer values) representing the tokenized sentence.
+The sequence structure is as follows (`<s>`, `</s>` and `<pad>` should be replaced by corresponding token IDs as specified by the dictionary):
+`<s>` + *tokenized sentence* + `</s>` + (`<pad>` tokens to pad to the maximum sequence length of 150)
+
+## Output
+
+name: preds
```
Comment (eaidova) [inline models/intel/machine-translation-nar-de-en-0001/description/machine-translation-nar-de-en-0001.md:37]:
```suggestion
name: `preds`
```

**Review by Anna-Mironova — APPROVED** [review]

**Pull request merged**